src/components/List/List.jsx

```diff
@@ -37,15 +74,27 @@ export const List = ({ token }) => {
     setFilterText(e.target.value);
   };
 
+  const setColor = (item) => {
```

Comment:
Another naming nit: I would call this `getColor` or instead of `setColor` because this function doesn't set anything, it just gives you a result based on what you give it. You might also consider something like `getCategory` or `getGroup` since the function itself doesn't have anything to do with colors.